tox.ini

```diff
@@ -35,55 +35,52 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps = 
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
+    django32: Django>=3.2,<4.0
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	python -Wd -m pytest {posargs}
+    python -Wd -m pytest {posargs}
 
 [testenv:docs]
 setenv = 
-	DJANGO_SETTINGS_MODULE = test_settings
-	PYTHONPATH = {toxinidir}
+    DJANGO_SETTINGS_MODULE = test_settings
+    PYTHONPATH = {toxinidir}
 whitelist_externals = 
-	make
-	rm
+    make
+    rm
 deps = 
-	-r{toxinidir}/requirements/doc.txt
+    -r{toxinidir}/requirements/doc.txt
 commands = 
-	doc8 --ignore-path docs/_build README.rst docs
-	rm -f docs/edx_django_utils.rst
-	rm -f docs/modules.rst
-	make -C docs clean
-	make -C docs html
-	python setup.py bdist_wheel
-	twine check dist/*
+    doc8 --ignore-path docs/_build README.rst docs
+    rm -f docs/edx_django_utils.rst
+    rm -f docs/modules.rst
+    make -C docs clean
+    make -C docs html
+    python setup.py bdist_wheel
+    twine check dist/*
 
 [testenv:quality]
 setenv = 
-	DJANGO_SETTINGS_MODULE = test_settings
+    DJANGO_SETTINGS_MODULE = test_settings
 whitelist_externals = 
-	make
-	rm
-	touch
+    make
+    rm
+    touch
 deps = 
-	-r{toxinidir}/requirements/quality.txt
+    -r{toxinidir}/requirements/quality.txt
 commands = 
-	touch tests/__init__.py
-	pylint edx_django_utils tests test_utils manage.py setup.py
-	rm tests/__init__.py
-	pycodestyle edx_django_utils tests manage.py setup.py
-	pydocstyle edx_django_utils tests manage.py setup.py
-	isort --check-only --diff tests test_utils edx_django_utils manage.py setup.py test_settings.py
-	make selfcheck
```

Comment:
This file has a lot of indentation changes. I guess you should undo them.

Reply:
In this Iteration we are accepting the changes made by `codemods` so that these are only one time changes and on the next iteration we wouldn't have any indentation related changes.